Applications/Terminal/Terminal.cpp

```diff
@@ -415,6 +476,42 @@ void Terminal::escape$J(const ParamVector& params)
     }
 }
 
+void Terminal::escape$S(const ParamVector& params)
+{
+    int count = 1;
+    if (params.size() >= 1)
+        count = params[0];
+
+    for (word i = 0; i < count; i++)
+        scroll_up();
+}
+
+void Terminal::escape$T(const ParamVector& params)
```

Comment:
Might be useful to add comments pointing to what part of the VT docs these are (SD for T, SU for S)

(Just a general comment, as it makes finding them to check what they are supposed to do easier)

Reply:
I think the best thing would be getting rid of the `escape$xxx` names and replacing them with names from the VT docs instead. :)
Not for this PR, but a nice future improvement.